### ISSUE TITLE
TalkBack doesn't read full page by "Read from top" or by item by item on COVID Alert menu page(shield)

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#606]
With TalkBack on, when the user is either going item by item (right swiping) or by "Read from top" to read full page for the COVID Alert menu screen/page, the TalkBack stops at the bottom of the visual screen (even if most of the text is cut off). The page doesn't automatically scroll. The user has to scroll the page up first and then continue reading. 

Also, if the user is at the bottom part of the page and select to read full page by "Read from top", it only starts from what's visually on the screen. User also can't go back up by swiping left.

Android 8.1.0 on Samsung Galaxy
Android 10 on OnePlus 7 (Font size and display size were at the largest size)

![image](https://user-images.githubusercontent.com/14280062/87165447-292f3a80-c298-11ea-9aa4-20208519452f.png)

![image](https://user-images.githubusercontent.com/14280062/87165805-a65aaf80-c298-11ea-90fa-2e2d2f2a17e5.png)

**Comment by nickmly** [comment]
Ref: #576

**Comment by JuliannaR** [comment]
@qanitah this issue will be fixed in the next build and has related issues you can find from Nick link above.

**Comment by timarney** [comment]
Closing this - as the PR was merged.  We can re-open as needed.
